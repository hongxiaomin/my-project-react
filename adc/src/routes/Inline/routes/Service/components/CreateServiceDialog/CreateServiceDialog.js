import React from 'react'
import propTypes from 'prop-types'
import DialogFilter from '../DialogFilter'

export const CreateServiceDialog = ({ isOpen, toggleDialog, info }) => (
  <DialogFilter
    DialogTitle='Init Service'
    isOpen={isOpen}
    toggleDialog={toggleDialog}
    info={info}
  />
)

CreateServiceDialog.propTypes = {
  info: propTypes.object.isRequired,
  isOpen: propTypes.bool.isRequired,
  toggleDialog: propTypes.func.isRequired,
}

export default CreateServiceDialog
