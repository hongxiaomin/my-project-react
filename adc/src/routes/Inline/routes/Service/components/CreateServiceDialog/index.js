import CreateServiceDialog from './CreateServiceDialog'

export default CreateServiceDialog
