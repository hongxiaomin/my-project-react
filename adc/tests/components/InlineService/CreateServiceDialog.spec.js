import React from 'react'
import { CreateServiceDialog } from 'routes/Inline/routes/Service/components/CreateServiceDialog/CreateServiceDialog'

describe('(Component) CreateServiceDialog', () => {
  const info = {}
  const isOpen = false
  const toggleDialog = jest.fn()
  it('render normally', () => {
    const wrapper = shallow(
      <CreateServiceDialog
        info={info}
        isOpen={isOpen}
        toggleDialog={toggleDialog}
      />,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
