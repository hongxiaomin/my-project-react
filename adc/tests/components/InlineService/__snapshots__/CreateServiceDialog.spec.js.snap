// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) CreateServiceDialog render normally 1`] = `
<Connect(DialogFilter)
  DialogTitle="Init Service"
  info={Object {}}
  isOpen={false}
  toggleDialog={[Function]}
/>
`;
